Automattic/newspack-plugin
# feat(ras): replace newsletters verification email

### All Submissions:






### Changes proposed in this Pull Request:

Replaces Newspack Newsletters' native email verification with RAS' account verification email when RAS is enabled.

This allows the verification email to take advantage of RAS' customization and the reader to verify their account in a different session/device, which is currently unsupported in Newspack Newsletters' implementation.

### How to test the changes in this Pull Request:

1. With RAS enabled and Newsletters configured with subscription lists, create a new account and verify the account's email
2. Delete the Newspack Newsletter's verification meta: `wp user meta delete {user_id} newspack_newsletters_email_verified`
3. Visit the "Newsletters" page of "My Account" and click to verify
4. Log out
5. Check the received email and confirm it's RAS' verification email
6. Visit the link in a fresh (unauthenticated) session and confirm you get authenticated, verified and redirected to the Newsletters page of "My Account"

### Other information:
